styles/components/alert.css

```diff
@@ -0,0 +1,46 @@
+.alert {
+  display: flex;
+  align-items: center;
+  border-radius: 2px;
+  padding: 0.5rem 1rem;
+  margin-bottom: 1rem;
+}
+.alert__text {
+  margin-left: 1rem;
+}
+.alert__close {
+  margin-left: auto;
+  opacity: 0.8;
+}
+
+.alert--info {
+  background-color: var(--info-primary);
+  border-left: 0.25rem solid var(--info-secondary);
+}
+.alert--info .alert__icon {
```

Comment:
any reason to use `-- `and `__` both the separators in classes?

Reply:
Yes, I was using BEM CSS, so `--` is for modifier and `__` for representing an element

Reply:
Plus the usage of 0.25rem is because I haven;t yet started with helper and utility classes, so once I add those, will make changes here as well.